test/unit/gateways/cyber_source_rest_test.rb

```diff
@@ -294,6 +294,22 @@ def test_authorize_apple_pay_visa
     end.respond_with(successful_purchase_response)
   end
 
+  def test_authorize_apple_pay_recurring
+    auth = @gateway.authorize(@amount, @apple_pay, @options)
+    @options[:stored_credential] = stored_credential(:merchant, :recurring, ntid: auth.network_transaction_id)
+    response = stub_comms do
+      @gateway.authorize(@amount, @apple_pay, @options)
+    end.check_request do |_endpoint, data, _headers|
+      request = JSON.parse(data)
+      assert_equal 'recurring', request['processingInformation']['commerceIndicator']
+      assert_equal 'merchant', request.dig('processingInformation', 'authorizationOptions', 'initiator', 'type')
```

Comment:
Looks great! Can you also add an assertion to ensure the cryptogram is not sent?
```suggestion
      assert_equal 'merchant', request.dig('processingInformation', 'authorizationOptions', 'initiator', 'type')
      assert_nil request.dig('paymentInformation', 'tokenizedCard', 'cryptogram')
```